cmd/root/root_cmd.go

```diff
@@ -77,5 +83,6 @@ func setFlags(rootCmd *cobra.Command) {
 	rootCmd.PersistentFlags().BoolVar(&debug, "debug", debug, "same as verbose but also show function names and line numbers")
 	rootCmd.PersistentFlags().BoolVar(&trace, "trace", trace, "enable trace logging")
 	rootCmd.PersistentFlags().DurationVar(&globalTimeout, "timeout", globalTimeout, "max overall execution duration")
+	rootCmd.PersistentFlags().StringVar(&logfile, "logfile", "", "file to write the logging output")
```

Comment:
Maybe: "file to write the logging output. If not specified logging output will be written to stderr"